package.json

```
@@ -166,7 +166,7 @@
     "qs": "6.9.4",
     "querystring-es3": "0.2.1",
     "react": "17.0.2",
-    "react-coin-icon": "0.1.44",
+    "react-coin-icon": "osdnk/react-coin-icon#06464588a3d986f6ef3a7d7341b2d7ea0c5ac50b",
```

Looks like this is already merged to `master` and skylar bumped version and published. 

```suggestion
    "react-coin-icon": "0.1.50",
```

(Obviously needs resolving yarn change too).